shop/src/main/resources/application-integrationtest.properties

```diff
@@ -0,0 +1,26 @@
+
+
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.H2Dialect
+
+spring.datasource.url=jdbc:h2:~/test;DB_CLOSE_ON_EXIT=FALSE;
```

Comment:
If this DB is for test, use an in-memory DB (jdbc:h2:mem:test)

Reply:
I think its fixed, i don't know why the request is not marked as outdated
Added the security too